### ISSUE TITLE
Make it more awesome!

### ISSUE THREAD

**Issue by zeroby0** [memefulsearch/memefulsearch.github.io#1]
Make it more awesome, it's not responsive right now,
take a look at [this](http://tympanus.net/codrops/2013/03/19/thumbnail-grid-with-expanding-preview/), here is the [demo](http://tympanus.net/Tutorials/ThumbnailGridExpandingPreview/)  
Awesome isn't it? You can make that "visit website" button to copy link.
And there is no feedback when there are no results, I thought the app froze, consider fixing it.  

9GAG brought me here :)

**Comment by anaganisk** [comment]
@zeroby0 what makes me use this rather than memeful is its simplicity and quicker loading even on mobiles, putting fancy animations will only spoil the ux atleast for mobile users

**Comment by zeroby0** [comment]
@anaganisk 
I said better, not slower or more fancy. You can make things a _lot_ better by compromising _very_ little. Most of the traffic the page is loading is from memeful, nothing we can do to speed it up or slow it down. Adding a little bit of css and making the site responsive improves the UX for everyone. The files need to be loaded only once and cached and there isn't even a noticeable change for a few more bytes. I made an entire ghost blog including all the fancy animations which loaded just as fast as this site (averaged at 600ms).

**Comment by anaganisk** [comment]
@zeroby0 You've got to be kidding me ryt? You say you've made a blog but you fail to realise that  the gif rendered from from memeful servers contribute to "memory and traffic" on mobile clients? you are still making redundant gif requests.
I dont understand what you mean making memeful load faster from their traffic. At the end of the days its on your device that it is rendered and that makes requests.
the site currently is good and responsive enough on mobile clients. IMHO it needs a good searchbar(visible and larger font) and better copy functionality. Anything more is just overkill, for the sole purpose of posting comments on 9gag, you have giphy for others.

**Comment by zeroby0** [comment]
oh god, learn to read.

**Comment by zeroby0** [comment]
dude, seriously? Read.
Most of the traffic is the gifs, a little bit of css won't slow down the website.
I didn't say anything about making _Memeful_ faster. Again, read.
And the gif requests are not redundant unless you repeat them unnecessarily.

**Comment by ennalikewine** [comment]
Hello, 
I have been reading the comments and I agree with both. 
There is room for grown but with the being said there will always be limitations. As ideas are fantastic there is always the challenge of making them a reality. To do this it will take time and energy from the GitHub community. If you would like to support the growth of memeful and memefulsearch, please contact me for future guidance and directives. It would appear that memefulsearch is looking to grow its database with the support and potential influence of 9GAG.
Please contact me for objectives and direction.
Please and thanks, 
Enna

**Comment by memefulsearch** [comment]
Hello @zeroby0, the demo looks good, but I dont really know how useful would be. Right now you only need to do one click to copy, if I add that peolple would have to make two click, one to open the item, and another to copy.

regarding the site not being responsive, well maybe it's not super beautiful, but it's using media queries so I would say it's responsive. Is there something incorrectly displayed on any of the devices you used?

**Comment by ennalikewine** [comment]
Hi @memefulsearch 

The site is good. What is your relationship like with 9GAG.com HQ and Memeful.com HQ?

I would like to have a better understanding of your intention with the site. 

How did you obtain the master list of the GIFs? I noticed you also have the rare retro GIFs that pertain to 08' 9GAG The "Hong Kong" Editions. 

I noticed Memeful.com has updated a few of the GIFs? Some have a larger source file, some move faster, some have been zoomed in, some have a better resolution as of Dec 2016.

I would imaging due to the source files having a larger MB size it has resulted in the Memefulsearch.github.io to run slightly slower. 

Great job! Thanks for the hard work and dedication.
> A ghost in the machine
